### ISSUE TITLE
Missing vocalization of the "Link copied" toast

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2000]
## Bug Report

**Problematic behavior**
When the "Copy link" button is activated with the keyboard, there is no vocal feedback from the toast confirming that the link has been copied.

**Expected behavior/code**
There must be an aria-live attribute present on the page for the message to be announced when the button is activated.

**Steps to Reproduce**
1. Log in to a document in read mode
2. Enable the screen reader
3. Navigate to the copy button with the keyboard
4. And then the bug happens!

**Possible Solution**
ARIA : [attribut aria-live](https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-live)
[Live regions ARIA : aria-live et ses analogues alert, log et status](https://access42.net/live-regions-aria-live-analogues-alert-log-status/)

**Additional context/Screenshots**

<img width="907" height="173" alt="Image" src="https://github.com/user-attachments/assets/869244ed-c0f4-46f3-9629-affc03538816" />

**Critères RGAA**

Critère 7.1 : [Chaque script est-il, si nécessaire, compatible avec les technologies d’assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact** 

Screen reader users are not informed of the confirmation of the action performed in the opened modal.
